x/incentives/keeper/gauge.go

```diff
@@ -477,6 +413,49 @@ func (k Keeper) GetEpochInfo(ctx sdk.Context) epochtypes.EpochInfo {
 	return k.ek.GetEpochInfo(ctx, params.DistrEpochIdentifier)
 }
 
+// AddToGaugeRewards adds coins to gauge.
+func (k Keeper) AddToGaugeRewards(ctx sdk.Context, owner sdk.AccAddress, coins sdk.Coins, gaugeID uint64) error {
+	if err := k.addToGaugeRewards(ctx, coins, gaugeID); err != nil {
+		return err
+	}
+
+	if err := k.bk.SendCoinsFromAccountToModule(ctx, owner, types.ModuleName, coins); err != nil {
+		return err
+	}
+	return nil
+}
+
+// addToGaugeRewards adds coins to gauge with the given ID.
+//
+// Returns error if:
+// - fails to retrieve gauge from state
+// - gauge is finished.
+// - fails to store an updated gauge to state
+//
+// Notes: does not do token transfers since it is used internally for token transferring value within the
+// incentives module or by higher level functions that do transfer.
+func (k Keeper) addToGaugeRewards(ctx sdk.Context, coins sdk.Coins, gaugeID uint64) error {
```

Comment:
Thoughts on naming this something like `addToGaugeRewardsNoSend` or `addToGaugeRewardsInternal` so someone doesn't accidentally call it in place of `AddToGaugeRewards` in the future?

Reply:
Oh I see you appended "internal" for tests – was there a specific reason you opted to not have it for the one here as well?

Reply:
To me, the fact that a method is unexported already reflects that it is internal.

The reason I added the "Internal" suffix in tests was because it clashed names with the exported variant